rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMEvent.java

```diff
@@ -39,7 +39,7 @@
  * @since ProActive Scheduling 0.9
  */
 @PublicAPI
-public class RMEvent implements Serializable, Cloneable {
+public class RMEvent implements Serializable, Cloneable, Comparable<RMEvent>, Unique {
```

Comment:
What is the Unique interface about?

Reply:
Objects which implement unique interface can be operated in SortedUniqueSet